Cargo.toml

```diff
@@ -18,3 +18,4 @@ hyper = "0.10"
 hyper-native-tls = "0.2.2"
 time = "0.1"
 chrono = "0.2"
+regex = "0.2"
```

Comment:
It might be better to use [url](https://crates.io/crates/url) for parsing DSNs instead.